lib/hawk.rb

```diff
@@ -1,8 +1,11 @@
 ##
 # Hawk entry point.
 #
+#
+
+require 'active_support/core_ext/array/wrap'
```

Comment:
I know, that's strange, but without this explicit `require` I get a
`#<NoMethodError: undefined method 'wrap' for Array:Class>`. I have the feeling it has to do with ActiveSupport's `require`-ing and autoloading.